templates/CRM/Price/Form/PriceSet.tpl

```diff
@@ -96,13 +96,13 @@
 
             {/if}
               {if !empty($extends) && $extends eq "Membership"}
-                {if (!empty($priceSet) && $element.id == $priceSet.auto_renew_membership_field) || (empty($priceSet) && $element.name == 'membership_amount')}
+                {if ($element.id == $priceSet.auto_renew_membership_field)}
```

Comment:
@eileenmcnaughton $element.id here is the price field id right?

Reply:
oh no - hmm - I mean - kinda but kinda no - I mean it SHOULD be price_field_value_id - but before it WAS just price_field_id I guess - arg.

I'll update to that & then follow up with a better fix